src/etc/inc/util.inc

```diff
@@ -1004,6 +1004,28 @@ function alias_expand($name) {
 	}
 }
 
+/* get description of alias */
+function get_alias_description($name)
```

Comment:
can we move this function into legacy_bindings.inc? it helps to identify the additions to the legacy code better.